src/VisualStudio/CSharp/Impl/project.lock.json

```diff
@@ -35,6 +43,14 @@
           "lib/net20/Microsoft.DiaSymReader.dll": {}
         }
       },
+      "NuGet.VisualStudio/3.3.0": {
```

Comment:
I think we need to get this out of the project.lock.json, and into the csproj with `EmbedInteropTypes`.